src/adapters/mc/asb/AzureLinuxBaseline.mof

```diff
@@ -1838,7 +1958,8 @@ instance of OsConfigResource as $OsConfigResource119ref
 
 instance of OsConfigResource as $OsConfigResource120ref
 {
-    ResourceID = "Ensure the isdnutils-base package is not installed";
+    ResourceID = "Ensure the isdn service is disabled and the isdnutils-base package is not installed";
+    ruleId = "51ebf409-911a-4d92-9d3a-1e331e7e4b27";
```

Comment:
nit: `ruleId` - should be PascalCase?

Reply:
Good catch! This works as is but yes, it should be PascalCase, this is a typo. Will fix.

Reply:
Fixed. Please see last iteration. Thanks for the feedback!